src/display.jl

```diff
@@ -209,6 +209,7 @@ Save a `Scene` with the specified filename and format.
 ## All Backends
 
 - `resolution`: `(width::Int, height::Int)` of the scene in dimensionless units (equivalent to `px` for GLMakie and WGLMakie).
+- `update`: Update the scene before saving.
```

Comment:
Maybe an explanation is needed what updating means (why would you not want it)

Reply:
Update the scene and its children (`update_limits!` and `center!`) before saving. One might want to set `update=false` when saving a zoomed scene.

Something like that ?